app/views/users/_users_table.html.erb

```diff
@@ -13,6 +13,7 @@
       </tr>
     </thead>
     <tbody>
+      <% if users != nil %>
```

Comment:
Kind of an idiomatic think but can you swap this out to `if users.present?` Want to take advantage of that activesupport!